src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -277,10 +279,16 @@ export class DatePickerMonth {
     const date = lastDate.getDate();
     const day = lastDate.getDay();
```

Comment:
Nitpick: For consistency, should this be `startDay` (similar to the naming in `getNextMonthDays`)?